api/src/test/java/uk/gov/hmcts/fees2/register/api/controllers/FeeControllerTest.java

```diff
@@ -186,34 +132,6 @@ public synchronized void searchFeeTest() throws Exception {
      */
     @Test
     public synchronized void searchFee_WithApprovedStatusTest() throws Exception {
-        feeCode = UUID.randomUUID().toString();
-        CreateRangedFeeDto rangedFeeDto1 = getRangeFeeDtoForSearch(0, 100, feeCode, FeeVersionStatus.approved, "civil money claims");
-        restActions
-            .withUser("admin")
-            .post("/fees-register/ranged-fees", rangedFeeDto1)
-            .andExpect(status().isCreated());
-
-        String feeCode2 = UUID.randomUUID().toString();
-        CreateRangedFeeDto rangedFeeDto2 = getRangeFeeDtoForSearch(101, 599, feeCode2, FeeVersionStatus.approved, "civil money claims");
-        restActions
-            .withUser("admin")
-            .post("/fees-register/ranged-fees", rangedFeeDto2)
-            .andExpect(status().isCreated());
-
-
-        String feeCode3 = UUID.randomUUID().toString();
-        CreateRangedFeeDto rangedFeeDto3 = getRangeFeeDtoForSearch(101, 699, feeCode3, FeeVersionStatus.draft, "probate");
-        restActions
-            .withUser("admin")
-            .post("/fees-register/ranged-fees", rangedFeeDto3)
-            .andExpect(status().isCreated());
-
-        String feeCode4 = UUID.randomUUID().toString();
-        CreateRangedFeeDto rangedFeeDto4 = getRangeFeeDtoForSearch(700, 999, feeCode4, FeeVersionStatus.pending_approval, "divorce");
-        restActions
-            .withUser("admin")
-            .post("/fees-register/ranged-fees", rangedFeeDto4)
-            .andExpect(status().isCreated());
 
         restActions
             .withUser("admin")
```

Comment:
Does this mean you're relying on the feeLoader.json fees over here?

Reply:
This is not FeeLoader test, this the actual fee creation using POST